src/workflows/orders.ts

```diff
@@ -7,7 +7,19 @@ import {
   OrdersApiCreateOrderRequest,
 } from '../api';
 import { serializeSignature, sign, signRaw } from '../utils';
+import { WalletConnection } from '../types/index';
 
+type CreateOrderWorkflowWithSignerRequest = WalletConnection & {
+  request: GetSignableOrderRequest;
+  ordersApi: OrdersApi;
+};
+
+type CancelOrderWorkflowWithSignerRequest = WalletConnection & {
```

Comment:
i thnk there should be interface

Reply:
can you tell me why it should be interface?